frontend/src/components/app/index.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```

Comment:
First thing to note is im using the index convention, which means if you name any file index then importing it from outside is simpler as you just import the containing folder. Please say if you dont like this convention. I like it for the the reason mentioned above and because it means if you hide away if there are containers.

Reply:
Ah on our last project the Lead thought us to move to the index convention. I think it's an interesting way to do it but it will have to be enforced! I'm happy for this to continue! I'll keep it in mind for future pull requests.